main/tests/server/src/com/google/refine/commands/project/SetProjectMetadataCommandTests.java

```diff
@@ -211,4 +213,51 @@ public void doPostThrowsIfCommand_getProjectReturnsNull() {
         verify(request, times(2)).getParameter("project");
         verify(projMan, times(1)).getProject(PROJECT_ID_LONG);
     }
+
+    @Test
+    public void setCustomMetadataTest() {
+        String customMetadata = this.getValidCustomMetadata();
+        ProjectMetadata meta = proj.getMetadata();
+        meta.setAnyField("customMetadata", customMetadata);
+
+        Assert.assertEquals(meta.getCustomMetadata("name"), "test");
+        Assert.assertEquals(meta.getCustomMetadata("isValid"), true);
+        Assert.assertEquals(meta.getCustomMetadata("id"), 1);
+        Assert.assertNotNull(meta.getCustomMetadata("companies"));
+        Assert.assertNotNull(meta.getCustomMetadata("information"));
```

Comment:
```suggestion
        Assert.assertTrue(meta.getCustomMetadata("companies") instanceof List);
        Assert.assertTrue(meta.getCustomMetadata("information") instanceof Map);
```
This will make the test a little bit stronger.

Reply:
this is great!  thank you 🙏